sdk/create-dapp/package.json

```diff
@@ -0,0 +1,61 @@
+{
+	"name": "@mysten/create-dapp",
+	"author": "Mysten Labs <build@mystenlabs.com>",
+	"description": "A cli for creating new SUi dApps",
+	"homepage": "https://sui-typescript-docs.vercel.app",
+	"version": "0.0.0",
+	"license": "Apache-2.0",
+	"files": [
+		"CHANGELOG.md",
+		"LICENSE",
+		"README.md",
+		"dist",
+		"src",
+		"templates"
+	],
+	"type": "commonjs",
+	"main": "./dist/cjs/index.js",
+	"module": "./dist/esm/index.js",
+	"types": "./dist/cjs/index.d.ts",
+	"exports": {
+		".": {
```

Comment:
Does this even need exports?

Reply:
probably not, just copied from an existing package. I'll remove these